Off-Top-App/off-top-flutter
# OT-navbarToTabs  | Replaced bottomNavigation with a DefaultTabController for better navigation efficiency

**What I Did:**

- Implemented a new dart file called tab.dart what uses DefaultTabController to allow a user to navigate through the app's pages

- Deprecated bottomNavigation.dart, router.dart, and routing_constants.dart as these functions aren't needed anymore

- Changed main.dart to have a home page of the login page

- Changed login.dart to redirect to the DefaultTabController

- Commented out any calls to the previous bottomNavigation.dart, namely in recordingSession.dart, and settings_page.dart

**How To Test:**

- You will need zookeeper, kafka, and spring boot running

- login into the application using any email already in your MySQL database

- You should be able to see the new DefaultTabController footer where the bottomNavigation footer used to be. It should look identical and functions should not have changed

- Test each tab: switch to Reports, then to Settings, then back to Home
